assets/html/scss/documenter/layout/_main.scss

```
@@ -125,6 +126,16 @@
       .docs-footer-nextpage {
         text-align: right;
       }
+
+      .flexbox-break {
+        flex-basis: 100%;
+        height: 0;
+      }
+
+      .footer-message {
+        font-size: 0.9em;
+        margin: 0.5em auto 0 auto;
```

Currently, if the footer goes over multiple lines it is left-aligned (e.g. on mobile). I'd make it centered:

```suggestion
        margin: 0.5em auto 0 auto;
        text-align: center;
```